en_US/faq/use-guide.md

```diff
@@ -417,3 +417,141 @@ Client disconnect link error code list:
+
+EMQX will ensure that messages with the same topic from the same client are forwarded in the order of their arrival, regardless of the QoS level of the messages. The QoS level does not affect the order of message forwarding. Whether messages are lost or duplicated, they will not cause message disorder. This is also the requirement from MQTT.
+
+However, EMQX does not guarantee the forwarding order of messages from different topics. We can consider them as entering different channels. For example, messages from topic A arrive at EMQX before messages from topic B, it is possible that messages from topic B will be forwarded earlier.
+
+## What should I do when I encounter problems related to client connection, publishing and subscribing?
+
+**Tags:** [*Debug*](tags.md#debug)
+
+EMQX's debug logs already record all the behaviors and phenomena. By reading the debug logs, we can know when the client initiated the connection, which parameters were specified during the connection, whether the connection was successful, and the reasons for connection rejection, etc. However, due to the excessive information recorded in debug logs, it may incur additional resource consumption and make it difficult to analyze individual clients or topics.
+
+So EMQX provides a [Log Trace](../getting-started/log.md) feature. We can specify the clients or topics we want to trace, and EMQX will output all the debug logs related to those clients or topics to the designated log file. This makes it convenient for both self-analysis and seeking help from the community.
+
+Just so you know, if the client cannot connect to EMQX for network reasons, the log tracing feature will not be helpful, because EMQX has not received any messages in such cases. This situation often occurs due to network configuration issues such as firewalls or security groups, which result in the server port not being open. This is particularly common when deploying EMQX on cloud instances. So, in addition to log tracing, we can troubleshoot network-related issues by checking port occupation, listening status, and network configurations.
+
+## Why are there client IDs like "CENSYS" or other unfamiliar clients?
+
+**Tags:** [*Security*](tags.md#security)
+
+CENSYS is an internet scanning and reconnaissance tool that periodically scans the IPv4 address space to discover default ports for protocols such as HTTP, SSH, MQTT, etc. So if you find MQTT clients with a client ID of "CENSYS" or other unknown clients accessing your MQTT broker, it indicates that you are currently at a relatively lower level of security protection. The following measures can effectively help you avoid this issue:
+
+1. Avoid using default configurations, such as the AppID and AppSecret used for verifying HTTP API access permissions in EMQX.
+2. Enable authentication, which can be password-based authentication or JWT authentication, to prevent situations where only knowledge of an IP address is required for login.
```

Comment:
```suggestion
2. Enable authentication mechanisms like password-based authentication or JWT authentication to prevent unauthorized access where only knowledge of an IP address is sufficient for login.
```